models/chainlink/arbitrum/chainlink_arbitrum_fm_fulfilled_transactions.sql

```diff
@@ -0,0 +1,65 @@
+{{
+  config(
+    tags=['dunesql'],
+    alias=alias('fm_fulfilled_transactions'),
+    partition_by=['date_month'],
+    materialized='incremental',
+    file_format='delta',
+    incremental_strategy='merge',
+    unique_key=['tx_hash', 'tx_index', 'node_address'],
+    post_hook='{{ expose_spells(\'["arbitrum"]\',
```

Comment:
what do you think about removing this post hook on all blockchain specific spells, then only apply on the cross-chain overall spell?

that way, the data explorer is less crowded, especially if you only plan to mostly query the cross-chain spells

if you have a strong opinion to keep as-is, that is fine too

Reply:
Yeah, not a problem.